plugins/woocommerce/src/Internal/Admin/ShippingLabelBanner.php

```diff
@@ -84,7 +73,7 @@ class_exists( '\Automattic\WCTax\Loader' );
 	 * @param \WP_Post $post Current post object.
 	 */
 	public function add_meta_boxes( $post_type, $post ) {
```

Comment:
I know this is untouched in the PR itself, but I don't believe we're receiving a `$post_type` parameter here, but rather the screen id (e.g. `woocommerce_page_wc-orders` if HPOS is enabled; it will be different for the legacy CPT edit page)